### ISSUE TITLE
DeepID accuracy might be wrong

### ISSUE THREAD

**Issue by pkchen53** [serengil/deepface#353]
You mention that DeepID got 97.05 accuracy scores on LFW data set. I believe you got the value from [here](https://github.com/Ruoyiran/DeepID),  but the model on this website is trained and tested using the YoutubeFaces dataset, not LFW.
Moreover,  according to the content mentioned in the [paper](https://arxiv.org/pdf/1406.4773.pdf) ```DeepID2 vectors are extracted by a total of 200 deep ConvNets, each of which is trained to extract two 160-dimensional DeepID2 vectors on one particular face patch and its horizontally flipped counterpart, respectively, of each face.```, there should be 200 independent ConvNets instead of just one like this [website](https://github.com/Ruoyiran/DeepID) to get a better accuracy. One can confirm the influence of the number of ConvNets from Figure9 of [DeepID](https://openaccess.thecvf.com/content_cvpr_2014/papers/Sun_Deep_Learning_Face_2014_CVPR_paper.pdf).

**Comment by serengil** [comment]
yes, you are right. I copied the model from the repo you've mentioned.

you should open an issue to the reference repo because I just wrapped it.
